docs/en/query_language/functions/hash_functions.md

```diff
@@ -30,11 +77,41 @@ Differs from sipHash64 in that the final xor-folding state is only done up to 12
 
 ## cityHash64
```

Comment:
As far as I remember it's some outdated version that was relevant during ClickHouse initial development, we'd better to point it out.